### PR TITLE
LL leftovers

### PR DESCRIPTION
**Pull request by cyocum**
This adds some secular genealogies in LL after Audacht Moraind.

### CODE REVIEW

File: LL/section_135.trig [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dctype: <http://purl.org/dc/dcmitype/> .
+@prefix irishRel: <http://example.com/earlyIrishRelationship.ttl#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<http://example.com/LL> {
+   <>
+        a dctype:Dataset;
+        dcterms:title "Section 135"@sga;
+        dcterms:isFormatOf <http://www.ucc.ie/celt/published/G800011F/text135.html>;
```
Comment (Eystein-Thanisch) [inline LL/section_135.trig]: I think CELT may have messed up. This URL points to Audacht Morainn itself (Section 134). Correct URL is https://celt.ucc.ie//published/G800011F/text136.html

File: LL/section_135.trig [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix foaf:  <http://xmlns.com/foaf/0.1/> .
+@prefix rel: <http://purl.org/vocab/relationship/>.
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix dctype: <http://purl.org/dc/dcmitype/> .
+@prefix irishRel: <http://example.com/earlyIrishRelationship.ttl#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+
+<http://example.com/LL> {
```
Comment (Eystein-Thanisch) [inline LL/section_135.trig]: Do we know anything about the scribal hand or textual history of this tract? And does it appear in O'Brien's edition? Perhaps we should group it in a separate sub-folder from the LL text of CGH, if there is a possibility that it is a distinct text.

File: LL/section_135.trig [hunk 3]
```diff
@@ -0,0 +1,397 @@
...
+
+<#Noenuail>
+        a foaf:Person;
+        irishRel:genName "Noenuail";
+        rel:childOf <#FhebriGlais>;
+        owl:sameAs <http://example.com/Rawl_B502/geneleach_osrithe.trig#Nóenail>.
+
+<#FhebriGlais>
+        a foaf:Person;
+        irishRel:genName "Fhebri Glais";
```
Comment (Eystein-Thanisch) [inline LL/section_135.trig]: irishRel:nomName "Faebar Glas";

File: LL/section_135.trig (hunk 2 above)
Comment (cyocum) [reply LL/section_135.trig]: Having a look, this is not in CGH, as CGH ends on f341 of LL.  Also, this is not CGH, this is the typed up version of Bergin and Best's diplomatic edition of LL (https://celt.ucc.ie//published/G800011F/header.html) so I would hesitate to equate it to CGH.

File: LL/section_135.trig (hunk 1 above)
Comment (cyocum) [reply LL/section_135.trig]: You are right. Good catch! I have fixed it.

**Comment by cyocum** [comment]
This is just a badly named branch. I was thinking that there was a lot of stuff after Audacht Morainn but from a cursory glance, the rest of the genealogies are saint ones or bishops.  They are not covered in CGH but I decided to add because it was pretty clear that this was secular.

I am unsure what "The LL "leftovers" haven't been fully added yet (Oengus Turbech? Cathair Már? Ailill Aulomm?)" means exactly.  This is the last file I think that will be added because the rest is religious unless you know of more secular ones in LL.

As for the sameAs, yes we can put more in later but remember that sameAs are symmetrical and transitive so they should get picked up.

**Comment by Eystein-Thanisch** [comment]
"The LL "leftovers" haven't been fully added yet (Oengus Turbech? Cathair Már? Ailill Aulomm?)" means that you haven't finished adding the pedigrees from this file. There are still three more to be added, unless there is a commit I have missed somewhere.

**Comment by cyocum** [comment]
I can confirm they are in the file and I have tracked them down in the web interface.  Maybe you are not seeing them because if the file is too big, the web interface collapses the file?  If you still cannot identify them, try checking out the branch and checking the file directly.

**Comment by Eystein-Thanisch** [comment]
Oh, yes so they are. The interface for reviewing the commit must have collapsed the file, as you say, although it didn't indicate that this had been done. Anyway, all good :-)

**Comment by cyocum** [comment]
Yeah, it is not a great interface in that regard. No worries. I will merge now.  Thanks for the review!

**Comment by Eystein-Thanisch** [comment]
No worries. I'll make sure I look at the actual branch in future such cases. Well done on getting all this material in so fast!

File: LL/section_135.trig (hunk 2 above)
Comment (Eystein-Thanisch) [reply LL/section_135.trig]: Whether or not O'Brien included in CGH might indicate whether it is normally part of that collection (e.g. maybe this section is part of CGH proper in the Book of Lecan or suchlike) or whether it is an independent tract. This is not a very relevant point, I suppose, as CGH's material is not itself internally harmonious! But we need to decide, for the purpose of queries, is this tract to have its testimony presented as part of that of LL's CGH or as a separate text (just like we have not made Accallam na Senórach part of the Laud genealogies proper, in terms of file structure).

Comment (Eystein-Thanisch) [reply LL/section_135.trig]: Does Schlueter mention it, by any chance?

Comment (cyocum) [reply LL/section_135.trig]: I think we should move this to email as I have closed this PR. I think it may get confusing to resurrect this PR.

**Pull request merged**